src/applications/personalization/profile/tests/components/personal-information/PersonalInformation.delete-email-address.unit.spec.jsx

```diff
@@ -35,19 +35,17 @@ function getEditButton() {
 
 // helper function that enters the `Edit contact email address` view and clicks on the `Remove` and `Confirm` buttons
 function deleteEmailAddress() {
-  const editButton = getEditButton();
-  editButton.click();
-
-  const emailAddressInput = view.getByLabelText(/email address/i);
-  expect(emailAddressInput).to.exist;
-
   // delete
-  view.getByText('Remove contact email address').click();
-  const confirmDeleteButton = view.getByText('Confirm', { selector: 'button' });
+  view
+    .getByLabelText(/remove contact email address/i, { selector: 'button' })
```

Comment:
did you have to switch from `getByText` to `getByLabelText`?

Reply:
for the same reason as above. The button text is remove